src/Concordium/Client/Types/Contract/Schema.hs

```diff
@@ -217,6 +265,14 @@ instance S.Serialize ContractSchemaV2 where
     pure ContractSchemaV2{..}
   put ContractSchemaV2 {..} = S.put cs2InitSig <> putMapOfWithSizeLen Four putText S.put cs2ReceiveSigs
 
+instance S.Serialize ContractSchemaV3 where
```

Comment:
Please add some tests cases to 
`/test/SimpleClientTests/SchemaParsingSpec.hs` for this.